app/src/test/java/com/x8bit/bitwarden/data/auth/datasource/network/service/IdentityServiceTest.kt

```diff
@@ -41,9 +40,7 @@ class IdentityServiceTest : BaseServiceTest() {
 
     private val identityService = IdentityServiceImpl(
         unauthenticatedIdentityApi = unauthenticatedIdentityApi,
-        json = Json {
-            ignoreUnknownKeys = true
-        },
+        json = json,
```

Comment:
This was the thing that started all of this. `json` is defined in the base test and uses the production Json instance.